### ISSUE TITLE
[BITV]: Sharing tab: permissions dropdown is not accessible 

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#43139]
![Screenshot from 2024-01-26 08-39-29](https://github.com/nextcloud/server/assets/6078378/06537315-a271-4dcb-9051-5ee3d5d15afd)

It is just a `<div>` with buttons inside, not a list ;(. Actually it should be a full `NcActions`, but don't know how to solve it without design changes.

**Comment by szaimen** [comment]
backport to stable28 is missing

**Comment by JuliaKirschenheuter** [comment]
closing as done

**Comment by szaimen** [comment]
Seems to work but I guess the seelection should always be on a new line?
![image](https://github.com/nextcloud/server/assets/42591237/f606e4b9-993c-42d0-9f9c-b5078f8ca049)

**Comment by ShGKme** [comment]
> Seems to work but I guess the seelection should always be on a new line?

- https://github.com/nextcloud/server/pull/43491
